pyairtable/api/api.py

```diff
@@ -17,17 +18,24 @@ class Api(ApiAbstract):
         >>> api.all('base_id', 'table_name')
     """
 
-    def __init__(self, api_key: str, timeout=None):
+    def __init__(
+        self,
+        api_key: str,
+        timeout=None,
+        max_records_per_request=DEFAULT_MAX_RECORDS_PER_REQUEST,
+    ):
         """
 
         Args:
             api_key: |arg_api_key|
 
         Keyword Args:
             timeout(``Tuple``): |arg_timeout|
-
+            max_records_per_request: |arg_max_records_per_request|
```

Comment:
Assuming my suggestions are accepted for the keyword-only argument changes, this snippet of docstring can also become an RST replacement for the keyword-only args accepted by the `AbstractApi` class, and reused in all 3 subclasses.